### ISSUE TITLE
new-win-policy not working for new window/tab links

### ISSUE THREAD

**Issue by Yonezpt** [nwjs/nw.js#6522]
NWJS: 0.29.1 SDK
OS: Win7 x64

Sample example used in the root of the NWJS folder

`package.json`
````json
{
    "name" : "sample",
    "main" : "main.html"
}
````
  
`main.html`  
````html
<!DOCTYPE html>
<html>
<head>
    <title>sample</title>
</head>
<body>
    <script type="text/javascript">
        nw.Window.get().on("new-win-policy", function(frame, url, policy) {
            policy.ignore();
            window.alert("link open action intercepted!");
        });
    </script>
    <a href="about:blank" target="_blank">This link will be blocked with normal click</a>
    <br/>
    <a href="about:blank" target="_blank">This link will be blocked with middle click</a>
    <br/>
    <a href="about:blank" target="_blank">This link will be blocked with ctrl + click</a>
    <br/>
    <br/>
    <a href="about:blank">This link will not be blocked if opened with middle click</a>
    <br/>
    <a href="about:blank">This link will not be blocked if opened with ctrl + click</a>
</body>
</html>
````

Open `nw.exe` and interact with each link following the described respective action.

The `new-win-policy` event intercepts the three first links correctly with the described action, but fails for the last two links using the described action.

`new-win-policy` should intercept the creation of any new tab/window regardless of the target origin/link action method.

**Comment by Yonezpt** [comment]
In addition to the above issue it appears that the event fails completely for any kind of link from within an iFrame event tho the event listener is set on the parent window.

**Comment by Christywl** [comment]
I can reproduce this issue on Linux/Windows with nwjs-sdk-v0.29.1.

**Comment by rogerwang** [comment]
This is fixed in git and will be available in the next [nightly build](http://dl.nwjs.io/live-build/).

**Comment by rogerwang** [comment]
@Yonezpt please file another issue for the additional problem.

**Comment by Yonezpt** [comment]
@rogerwang Understood, I will wait for the release of the first fix and see if it happens to also fix the iFrame issue I mentioned, otherwise I will do as you said.

**Comment by Yonezpt** [comment]
@rogerwang Happy to report that the fix in 0.29.2 for the main issue also covers the similar one with link clicks originated from inside iFrames, all new window open attempts are being intercepted correctly now.
